examples/spark/src/main/scala/org/apache/carbondata/examples/CarbonExample.scala

```diff
@@ -44,7 +44,7 @@ object CarbonExample {
     // Currently there are two data loading flows in CarbonData, one uses Kettle as ETL tool
     // in each node to do data loading, another uses a multi-thread framework without Kettle (See
     // AbstractDataLoadProcessorStep)
-    // Load data with Kettle
+    // Load data without Kettle(by Default,use_kettle='false')
```

Comment:
Thank you for your reply.
But this example really messed me up at the beginning,making me believe that use kettle or not can cause different query results.